src/DotNetty.Codecs.Redis/Messages/IBulkStringRedisContent.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DotNetty.Codecs.Redis.Messages
+{
+    using DotNetty.Buffers;
+
+    public interface IBulkStringRedisContent : IRedisMessage, IByteBufferHolder
+    {
+        bool IsNull { get; }
```

Comment:
It is actually from https://github.com/netty/netty/blob/4.1/codec-redis/src/main/java/io/netty/handler/codec/redis/BulkStringRedisContent.java